### ISSUE TITLE
Can sound size be filtered?

### ISSUE THREAD

**Issue by xiaoqiang1999** [ricky0123/vad#83]
Hello! This library has been very helpful to us. Thank you and your team for their work.

But I encountered a question, can it filter according to the size of the sound? Is there a filtering threshold for sound size?

If not, would you consider adding this feature?

Thank you for your reply.

**Comment by kyb3r** [comment]
Do you mean filtering for minimum speech length?

**Comment by xiaoqiang1999** [comment]
> Do you mean filtering for minimum speech length?

No, I mean the loudness of the sound, filtering out the sound below a certain loudness.
But it seems that this can be achieved indirectly by configuring positiveSpeedThreshold and negativeSpeehThreshold?
